.babelrc

```diff
@@ -0,0 +1,13 @@
+
+{
+  "presets": ["es2015-node5"],
```

Comment:
It looks like node5 will be a requirement to use this package.  That's ok with me, I've been wanting an excuse to update!